fixtures/GeneralStateTests/stPreCompiledContracts/current_mnt_id.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
1. follow #659 to put this newly created test case json to `QuarkChainStateTests` folder.
2. file naming should be camelCase
3. fix `_info`
4. remove unused `post` states and only keep `ConstantinopleFix`